src/dali/plugin/input/csv/blockfi.py

```diff
@@ -111,7 +137,25 @@ def load(self) -> List[AbstractTransaction]:
                             crypto_received=line[self.__AMOUNT_INDEX],
                         )
                     )
-                elif line[self.__TYPE_INDEX] == _WITHDRAWAL:
+                elif entry_type == _ACH_WITHDRAWAL:
+                    last_withdrawal_fee = None
+                    result.append(
+                        OutTransaction(
+                            plugin=self.__BLOCKFI,
+                            unique_id=Keyword.UNKNOWN.value,
+                            raw_data=raw_data,
+                            timestamp=f"{line[self.__TIMESTAMP_INDEX]} -00:00",
+                            asset=line[self.__CURRENCY_INDEX],
+                            exchange=self.__BLOCKFI,
+                            holder=self.account_holder,
+                            transaction_type="Sell",
+                            spot_price=Keyword.UNKNOWN.value,
+                            crypto_out_no_fee=str(-RP2Decimal(line[self.__AMOUNT_INDEX])),
+                            crypto_fee="0",
+                            notes="ACH redrawal",
```

Comment:
It's "ACH withdrawal". But again, I think referral to ACH is mostly confusing to the final user: remove the note altogether?